### DISCUSSION TITLE
Update product quantities from Quote/Invoice

### DISCUSSION THREAD

**Discussion by deepti4666** [YetiForceCompany/YetiForceCRM#14358] [How to] [answered]
Hi,

I have created 2 new fields
Quantity in Demand - to be updated from Quotes
Quantity Sold - to be updated from Invoices

I want to update the product quantities from the related quotes/invoices so I can see what was sold/in demand

How do I do this, I have tried a workflow via update-related record - but there is nothing linking it to those fields.

Please help

**Comment by vovpff** [comment, marked as answer]
You have 2 ways:
Self-written handler what do calculations
Some ready to run extension
